src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  box-sizing: border-box;
```

Comment:
Move to * selector and remove it from all other places.

Reply:
fixed